internal/app/feed/feed_group.go

```diff
@@ -75,8 +75,18 @@ func (f *GroupFeed) Export(ctx context.Context, apiClient api.APIClient, exporte
 		util.Check(err, "Failed to unmarshal data to struct")
 
 		if len(rows) != 0 {
-			err = exporter.WriteRows(f, rows)
-			util.Check(err, "Failed to write data to exporter")
+			// Calculate the size of the batch we can insert into the DB at once. Column count + buffer
```

Comment:
I really wonder if there is a way to put all this together, since it's repeated code essentially everywhere. Maybe have the logic in `exporter.WriteRows`?

Reply:
Unfortunately not without a lot more code. Since we are relying on reflection for inserting into SQL Databases we can't easily pass a slice of a map or pass the data to a standard struct.